dist/scss/wds-variables/_typography.scss

```diff
@@ -21,8 +22,33 @@ $wds-typescale-size-plus-7: 54px;
 $wds-fandom-font-family: Rubik, Helvetica, Arial, sans-serif;
 
 // Fandom font weights
+// @deprecated, use $wds-font-weight- instead
 $wds-fandom-font-weight-light: 300;
 $wds-fandom-font-weight-regular: 400;
 $wds-fandom-font-weight-medium: 500;
 $wds-fandom-font-weight-bold: 700;
 $wds-fandom-font-weight-black: 900;
+
+// Font weights
+$wds-font-weight-light: 300;
```

Comment:
@xkxd can you use this variable within .wds-font-weight-light class? and the same apply for other vars